backbone/src/main/java/org/researchstack/backbone/model/survey/SubtaskQuestionSurveyItem.java

```diff
@@ -10,27 +10,4 @@
     SubtaskQuestionSurveyItem() {
```

Comment:
I haven't had a chance to look at Survey in iOS and how it relates to Question and (sub)Task, but this name makes it sounds complex!